routing/datasource.go

```diff
@@ -97,6 +118,9 @@ func mergeDefs(defsByClient map[DataClient]routeDefs) []*eskip.Route {
 	return all
 }
 
+// receives the initial set of the route definitiosn and their
+// updates from multiple data clients, merges them by route id
+// and sends the merged route definitions to the output channel.
```

Comment:
With this implementation we can't define data source priorities, correct? It's never deterministic which one will win for the same route id?